setup.py

```diff
@@ -22,7 +22,7 @@
 ]
 
 extras_require = {
-    "format" : ["rfc3987", "strict-rfc3339", "webcolors"],
```

Comment:
I think probably we'd need to support backwards compat here too. I assume you've already verified that strict-rfc3339 and webcolors are non-GPL?

If so I think we'd need to add a format-nongpl extra, otherwise we'll potentially change the deps being installed for people.